src/frontegg-oauth-client.ts

```diff
@@ -155,15 +156,64 @@ const isTokenExpired = (tokenExpirationTime: number | null) => {
   return tokenExpirationTime - 60 * 60 * 1000 < Date.now()
 }
 
-const GET_FRONTEGG_USER_DATA_RESPONSE_SCHEMA = z.object({
-  id: z.string(),
+/**
+ * Function to decode a JWT token
+ *
+ * @param token the JWT token to decode
+ * @returns the decoded JWT token
+ */
+const decodeJwt = (token: string) => {
```

Comment:
❓ Did you come up with this algorithm? or is it copied from some reference?

Reply:
Combination of the referenced stackoverflow and https://github.com/auth0/jwt-decode/blob/main/lib/index.ts. Will add a mention of that library as well.